README.md

```diff
@@ -16,7 +16,15 @@ The first two sources are set by the lein-environ and boot-environ
 plugins respectively, and should not be edited manually.
 
 The `.lein-env` file is populated with the content of the `:env` key
-in the Leiningen project map.
+in the Leiningen project map. Keywords with a `project` namespace are
+looked up in the project map. For example:
+
+```clojure
+{:env {:app-version :project/version}}
+```
+
+This looks up the `:version` key in the Leiningen project map. You can view
+the full project map by using [`lein-pprint`](https://github.com/technomancy/leiningen/tree/master/lein-pprint).
```

Comment:
lein-pprint shoudn't be in code quotes, as it's the name of a library, rather than a command.